### PR TITLE
Support sqlite to postgres db import

### LINKED ISSUE #565: Fix db import for apps/dashboards sqlite ->pg
Importing existing sqlite db with bookmarks fails when targeting Postgres.

### PR DESCRIPTION
**Pull request by suprjinx**
When importing a sqlite database to postgres, boolean and UUID fields for Apps and Dashboards need to be translated.

This PR extends the "translateFields" function to support UUID type and an additional boolean field.

Fixes #565

### CODE REVIEW

File: tests/integration/golang/database/import_test.go [hunk 1]
```diff
@@ -58,37 +58,72 @@ func (s *ImportTestSuite) SetupTest() {
 	assert.Nil(s.T(), database.CreateDefaultExperiment(db.GormDB(), "s3://fasttrackml"))
 	s.inputDB = db.GormDB()
 
-	inputExperimentFixtures, err := fixtures.NewExperimentFixtures(db.GormDB())
-	assert.Nil(s.T(), err)
-	inputRunFixtures, err := fixtures.NewRunFixtures(db.GormDB())
+	inputRunFixtures, err := fixtures.NewRunFixtures(s.inputDB)
 	assert.Nil(s.T(), err)
 	s.inputRunFixtures = inputRunFixtures
+	s.populateDB(s.inputDB)
```
Comment (dave-gantenbein) [inline tests/integration/golang/database/import_test.go:64]: Does this test specifically demonstrate the fix to the reported problem? If it's must my inability to grok these verbose golang tests then just point it out and tell me to go away. :)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -125,6 +125,7 @@ jobs:
         env:
           DOCKER_BUILDKIT: 1
           FML_DATABASE_URI: ${{ matrix.database-uri }}
+          FML_OUTPUT_DATABASE_URI: ${{ matrix.database-uri }}
```
Comment (suprjinx) [inline .github/workflows/ci.yml:128]: this will cause import_test to use the parameter as the destination (so we cover importing to postgres).

**Review by dave-gantenbein — APPROVED** [review]
LGTM

**Review by dave-gantenbein — APPROVED** [review]

**Review by fabiovincenzi — DISMISSED** [review]
lgtm

**Review by dave-gantenbein — APPROVED** [review]
lgtm

**Pull request merged**